src/include/homeobject/blob_manager.hpp

```diff
@@ -10,7 +10,7 @@
 namespace homeobject {
 
 ENUM(BlobError, uint16_t, UNKNOWN = 1, TIMEOUT, INVALID_ARG, NOT_LEADER, UNKNOWN_SHARD, UNKNOWN_BLOB, CHECKSUM_MISMATCH,
-     READ_FAILED, INDEX_ERROR, SEALED_SHARD);
+     READ_FAILED, INDEX_ERROR, SEALED_SHARD, TOMBSTONE_BLOB, ALIVE);
```

Comment:
These are publicly visible but will never be used; we shouldn't add them here or all our consumers are forced to add `case`s for them that are meaningless and can never happen.
From the applications perspective the only things it is concerned with is whether the BLOB did not exist on a READ. Delete on a deleted blob is idempotent and immediate success. So we shouldn't need any new errors in the public API.

Reply:
If we find we do need to add something here (because some external client needs this information) we need to make a corresponding error in the `nuobject_proto` repo so we can return this value from SM; we can bump the version to `0.14.1` here so we don't break SM in the meantime (it's locked to 0.13.x which 0.14.0 is consider part of oddly)

Reply:
make sense